internal/instantRunoff/instantRunoff.go

```diff
@@ -73,17 +130,18 @@ func Run(votes []*Vote, nameMap map[int]string) string {
 					n += 1
 				}
 			}
+			for _, tally := range tallies {
+				if tally.ID == eliminatedID {
+					tally.Eliminated = true
+				}
+			}
```

Comment:
I'd pull this one level up so it's not in the same scope as n, since the only reason that this block is its own block is to show that the whole thing from line 126 to 138 is one operation, ie. filter ids.